### PR TITLE
[TRAFODION-3070] we cannot comment on native hive table.

### PR DESCRIPTION
**Pull request by zlei929**
For now, i registered them in traf metadata before do 'comment' operations on native HIVE tables, and displayed comments when do 'SHOWDDL' operations on them.

### CODE REVIEW

**Comment by DaveBirdsall** [comment]
jenkins, add user

**Comment by DaveBirdsall** [comment]
Release build failed because it could not access some jar. Likely a network glitch.

File: core/sql/bin/SqlciErrors.txt [hunk 1]
```diff
@@ -515,6 +515,7 @@
 1718 ZZZZZ 99999 BEGINNER MINOR DBADMIN --- unused ---
 1719 ZZZZZ 99999 BEGINNER MINOR DBADMIN Access Type '$0~string0' is not supported.
 1720 ZZZZZ 99999 BEGINNER MINOR DBADMIN Isolation Level '$0~string0' is not supported.
+1721 ZZZZZ 99999 BEGINNER MINOR DBADMIN Cannot comment on '$0~string0' if this is a native hive table.
```
[the remark this answers is not in the document]
Comment (zlei929) [reply core/sql/bin/SqlciErrors.txt]: Thanks, and i'm sorry that i did not see this document(https://cwiki.apache.org/confluence/display/TRAFODION/How+to+Add+A+New+SQL+Error+Diagnostic)  until I pull this request,  i will solve  this  problem in the next pull request.

File: core/sql/sqlcomp/CmpSeabaseDDLcommentOn.cpp [hunk 2]
```diff
@@ -229,6 +229,16 @@ void  CmpSeabaseDDL::doSeabaseCommentOn(StmtDDLCommentOn   *commentOnNode,
   enum ComObjectType enMDObjType = COM_UNKNOWN_OBJECT;
 
   ComObjectName objectName(commentOnNode->getObjectName());
+  //we can not comment on native hive table.
+  if (str_cmp(toUpper(currCatName.data()), "HIVE", 4) == 0)
```
[the remark this answers is not in the document]
Comment (zlei929) [reply core/sql/sqlcomp/CmpSeabaseDDLcommentOn.cpp]: Thank you for your reminding~!   i will create a new pull request after changed this.

**Comment by anoopsharma00** [comment]
Some of the comments in TRAFODION-3084 apply here as well. 

Were testcases added to regressions to test this feature?
Would 'drop table hive.hive.t' or 'cleanup table hive.hive.t' command remove added comments?
Would unregister command remove comments? It will remove the added objectuid for that hbase table.
It will also be better to use 'if not registered' instead of 'if not exists' construct in the register command.
They both are supported.

Another question: it will be better that comments are added to NATable structure when it is
constructed. Right now looks like comments are read from metadata whenever a showddl command is
issued. That is an expensive operation and will be repeated for each call.
Comments should be read when NATable is constructed and then attached to it. This is how all
other table related structures/data are populated and kept. Those comments can then be retrieved
from NATable instead of reading from metadata.

**Pull request merged**